parlai/clib/cuda/ngram_repeat_block_cuda_kernel.cu

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+This source code is licensed under the MIT license found in the
```

Comment:
And what @pearlli98 mentioned about licensing should apply to this file as well, I suppose?